index.html

```diff
@@ -483,6 +521,20 @@ <h2>
         </table>
       </section>
     </section>
+    <section>
+      <h2>
+        Feature Policy integration
+      </h2>
+      <p data-link-for="Navigator">
+        The Battery Status API is a <a>policy-controlled feature</a>, as
+        defined by Feature Policy [[!FEATURE-POLICY]]. The <a>feature name</a>
+        for the Battery Status API is "<code>battery</code>". The <a>default
+        allowlist</a> for the Battery Status API is <code>« "self" »</code>.
```

Comment:
Minor nit: the value for the default allowlist would be `'self'`, with single quotes included

You can just say now

> The Battery Status API is a policy-controlled-feature identified by the string "`battery`". It's default allowlist is `'self'`.

Reply:
Ditto, pushed an update per your proposal.